File: source/repos/BlackJack/BlackJack.ViewModels/GameViews/EndGameView.cs
using BlackJack.DataAccess.Enums;
using System.Collections.Generic;

namespace BlackJack.ViewModels.GameViews
{
    public class EndGameView
    {
        public StatusType Status { get; set; }
        public string Winner { get; set; }
        public PlayerEndGameView Player { get; set; }
        public List<BotEndGameViewItem> Bots { get; set; }

        public EndGameView()
        {
            Bots = new List<BotEndGameViewItem>();
            Player = new PlayerEndGameView();
        }
    }
    public class PlayerEndGameView
    {
        public string Name { get; set; }
        public List<CardEndGameViewItem> Cards { get; set; }
    }
    public class BotEndGameViewItem
    {
        public string Name { get; set; }
        public List<CardEndGameViewItem> Cards { get; set; }
    }

    public class CardEndGameViewItem
    {
        public CardRankType Rank { get; set; }
        public CardSuitType Suit { get; set; }
    }
}

File: source/repos/BlackJack/BlackJack.WEB/ClientApp/src/app/shared/entities/game/end-game.view.ts
import { CardRankType } from 'src/app/shared/enums/card-rank-type.view';
import { CardSuitType } from 'src/app/shared/enums/card-suit-type.view';
import { StatusType } from 'src/app/shared/enums/status-type.enum.view';

export class EndGameView {
    status: StatusType;
    winner: string;
    player:PlayerEndGameView;
    bots:BotEndGameViewItem[] = [];
}
export class PlayerEndGameView{
    name:string;
    cards:CardEndGameViewItem[] = [];
}
export class BotEndGameViewItem{
    name:string;
    cards:CardEndGameViewItem[] = [];
}
export class CardEndGameViewItem{
    rank:CardRankType;
    suit:CardSuitType;
}